### ISSUE TITLE
Introduce global attributes "license" and "copyright"?

### ISSUE THREAD

**Issue by rjhogan** [cf-convention/discuss#294]
I am planning to release some of the static fields used by the ECMWF model (e.g. aerosol and albedo climatologies) in netCDF format, and would like to express the copyright and the license in the global attributes of the file.  This would seem to me good practice, just as it is would be when releasing open-source software, and indeed for data Zenodo requires the license to be stated.  But I can't find any reference to "license" or "copyright" global attributes in the CF conventions - are there plans to add these?  I guess we could have license="CC-BY-4.0", and copyright="2024 ECMWF" but it would be nice to follow a convention if there is one. Note that the CC-BY-4.0 license requires the licensee to acknowledge the creator and copyright holder of the data (if provided) in any downstream products, so the ability to state the creator/copyright-holder in netCDF metadata would go together with the ability to state the license.
Robin Hogan (ECMWF)

**Comment by ajelenak** [comment]
The [Attribute Convention for Data Discovery](https://wiki.esipfed.org/Attribute_Convention_for_Data_Discovery_1-3) has the `license` attribute and is designed to fit nicely within the CF conventions. Also has several attributes for data creator.

**Comment by taylor13** [comment]
@ajelenak I was not aware of these conventions.  Do you know whether they have been adopted by any large projects?  What advantages would there be in the next phase of CMIP to changing the names of some of the global attributes used in past CMIP phases to be consistent with the Data Discovery dimensions.

**Comment by ajelenak** [comment]
It is strongly recommended for all NASA satellite data, but soon it will be required. It is also recommended for NOAA satellite data (included in the NCEI's file templates).

**Comment by rmendels** [comment]
Most gridded datasets in the many ERDDAP servers have these conventions in their metadata,  and that is a lot of datasets.  I believe,  but could be wrong,  that most NOAA Coastwatch and NOAA IOOS datasets use these conventions.

**Comment by rmendels** [comment]
I should add that ERDDAP allows you to "add" metadata to existing datasets without having to modify existing files.  That is why many ERDDAP datasets have these metadata even though the underlying files may not.

**Comment by rjhogan** [comment]
Thank you for your comments!  It is surprising that no-one has thought to add a "copyright" global attribute when most data licenses (e.g. CC-BY-4.0, clause 3.a.1.A.ii) explicitly require that copyright statements are retained when sharing modified versions of a dataset. The copyright holder is usually the institution rather than the individual creator. Perhaps I should contact the ACDD people, but it is less clear how to do this.

**Comment by ajelenak** [comment]
> Perhaps I should contact the ACDD people, but it is less clear how to do this.

I'm one of them. @ethanrd is another, and the creator of initial ACDD.

**Comment by bzah** [comment]
@taylor13 > Does anyone else have any experience with datasets conforming to the [Attribute Convention for Data Discovery](https://wiki.esipfed.org/Attribute_Convention_for_Data_Discovery_1-3)? 

Eumetsat's MTG products also comply with the ACDD convention.

**Comment by MathewBiddle** [comment]
NCEI's netCDF templates build off CF and ACDD. https://www.ncei.noaa.gov/netcdf-templates

**Comment by cofinoa** [comment]
For the record,

I would suggest that CF users adopt the ACDD conventions to complement CF metadata, rather than CF defining new attributes in parallel.

As a concrete example, the IPCC ATLAS intermediate dataset at the Copernicus Climate Change Service already applies ACDD 1.3:
https://cds.climate.copernicus.eu/datasets/multi-origin-c3s-atlas

**Comment by MathewBiddle** [comment]
TLDR; stick your license in a `license` attribute and your copyright in a `copyright` attribute. 

For examples, a majority of the data found in the IOOS Data Catalog follow ACDD-1.3: https://data.ioos.us/dataset/.

Most of those datasets have been checked against ACDD-1.3 using the IOOS Compliance Checker: https://compliance.ioos.us/index.html

Agreed. This should be an ACDD recommendation, not CF. 

As @rmendels mentioned above, if sharing data with ERDDAP, there is a setting to identify the license for the data. That information is populated in the `license` attribute. While that attribute is not part of the defined ACDD recommendations, it seems to be quite ubiquitous, given the heavy use of ERDDAP in the ocean community. This addition of the attribute doesn't break any of the standards/recommendations. 

There is not a clear path on how to update ACDD to the next iteration. It was managed through ESIP, but seems like the leads have moved on to other activities. If someone has the capacity and interest, it could be due for an update.

**Comment by JonathanGregory** [comment]
Should the CF standard include a recommendation to use ACDD for discovery metadata?

**Comment by feggleton** [comment]
Hi all, Apologies I had missed this discussion. I agree to use ACDD and this is not something for CF to add, but actually point to ACDD as a recommendation for discovery metadata in files. We have a Roadmap section about discovery metadata so this could be the statement in the roadmap document that 'CF will include a recommendation to use ACDD for discovery metadata' - I think that would be sufficient. Here is the roadmap discussion: https://github.com/orgs/cf-convention/discussions/394 - I will be happy to write something into the draft roadmap document once created.
